tests/Doctrine/Tests/Inflector/Rules/Dutch/DutchFunctionalTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Inflector\Rules\Dutch;
+
+use Doctrine\Inflector\Inflector;
+use Doctrine\Inflector\InflectorFactory;
+use Doctrine\Inflector\Language;
+use Doctrine\Tests\Inflector\Rules\LanguageFunctionalTest;
+
+class DutchFunctionalTest extends LanguageFunctionalTest
+{
+    /**
+     * @return string[][]
+     */
+    public function dataSampleWords(): array
+    {
+        return [
+            ['weerman', 'weermannen'],
+            ['ei', 'eieren'],
+            ['rij', 'rijen'],
+            ['mogelijkheid', 'mogelijkheden'],
+            ['adres', 'adressen'],
+            ['olie', 'oliën'],
+            ['industrie', 'industrieën'],
+            ['lid', 'leden'],
+            ['smid', 'smeden'],
+            ['schip', 'schepen'],
+            ['stad', 'steden'],
+            ['gelid', 'gelederen'],
+            ['gelid', 'gelederen'],
```

Comment:
This is a duplicate, please sort that array so that we can spot other duplicates if any

Reply:
I did it for you